src/main/resources/citizen/templates/document/defendantResponseReceipt.html

```diff
@@ -236,17 +236,21 @@
       <td colspan="2"><p><strong>Claim number: </strong>{{ claim.referenceNumber }}</p></td>
     </tr>
     <tr>
-      <td colspan="2"><p><strong>Issued on: </strong> {{ claim.issuedOn }}</p></td>
+      <td colspan="2"><p><strong>Issued on: </strong>
+        <span id="timestamp">{{ claim.issuedOn }}</span>
```

Comment:
`id`s should be unique in a document. Consider using a `class` instead.

Reply:
Next changes uses data-type instead of id